features/base/tracks/reducer.js

```diff
@@ -1,14 +1,30 @@
 import { ReducerRegistry } from '../redux';
 
-import { TRACK_ADDED, TRACK_REMOVED } from './actionTypes';
+import {
+    TRACK_ADDED,
+    TRACK_MUTE_CHANGED,
+    TRACK_REMOVED
+} from './actionTypes';
+
+import {
+    TOGGLE_AUDIO_MUTED_STATE,
+    TOGGLE_VIDEO_MUTED_STATE
+} from '../../toolbar/actionTypes';
+
 
 /**
  * Listen for actions that add or remove remote tracks.
  */
 ReducerRegistry.register('features/base/tracks', (state = [], action) => {
     switch (action.type) {
+    case TOGGLE_AUDIO_MUTED_STATE:
```

Comment:
Yes, it was to trigger an update for the local state change.

+1 to adding audio/videoMuted to participants